index.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Learn GitHub</title>
+</head>
+<body>
+	<p>Just a throwaway index.html page.</p>
```

Comment:
You should add an h1 element.  But you already know that, because you're the one who is writing this comment!